packages/react/src/evm/components/ConnectWallet/index.tsx

```diff
@@ -63,6 +63,7 @@ interface ConnectWalletProps extends ThemeProviderProps {
     loginOptional?: boolean;
   };
   className?: string;
+  btnTitle?: React.ReactNode;
```

Comment:
I believe this should be `JSX.Element`?

Reply:
or maybe `JSX.Element | string` ?

Reply:
I forgot the difference between`JSX.Element` and `React.ReactNode`, so I checked. 

 `JSX.Element | string` sounds good because it will accept only one element. No arrays will be passed, and it may be just a string. I will update it now